### PR TITLE
auth.html redirect sample

### PR DESCRIPTION
**Pull request by sameerag**
Sample to demonstrate `redirectUri` as a blank html for `acquireTokenSilent()` calls.

### CODE REVIEW

File: samples/VanillaJSTestApp/index_blankPageRedirectUri.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Quickstart for MSAL JS</title>
```
Comment (jasonnutter) [inline samples/VanillaJSTestApp/index_blankPageRedirectUri.html]: Can you make this unique for this sample?

File: samples/VanillaJSTestApp/index_blankPageRedirectUri.html [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    <script type="text/javascript" src="https://alcdn.msauth.net/lib/1.1.3/js/msal.js" integrity="sha384-m/3NDUcz4krpIIiHgpeO0O8uxSghb+lfBTngquAo2Zuy2fEF+YgFeP08PWFo5FiJ" crossorigin="anonymous"></script>
+    <script type="text/javascript">
+        if(typeof Msal === 'undefined')document.write(unescape("%3Cscript src='https://alcdn.msftauth.net/lib/1.1.3/js/msal.js' type='text/javascript' integrity='sha384-m/3NDUcz4krpIIiHgpeO0O8uxSghb+lfBTngquAo2Zuy2fEF+YgFeP08PWFo5FiJ' crossorigin='anonymous'%3E%3C/script%3E"));
+    </script>
+</head>
+
+<body>
+    <div class="container">
+        <div class="leftContainer">
+            <p id="WelcomeMessage">Welcome to the Microsoft Authentication Library For Javascript Quickstart</p>
```
Comment (jasonnutter) [inline samples/VanillaJSTestApp/index_blankPageRedirectUri.html]: Can you add a note about what this sample is demonstrating?

File: samples/VanillaJSTestApp/server.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ app.use("/dist", express.static(path.join(__dirname, "../../lib/msal-core/dist")
 
 // Set up our one route to the index.html file.
 app.get('*', function (req, res) {
-    const reqPath = req.path === "/" ? "/index.html" : req.path;
+    const reqPath = req.path === "/" ? "/index_blankPageRedirectUri.html" : req.path;
```
Comment (jasonnutter) [inline samples/VanillaJSTestApp/server.js]: I don't agree with this change, the existing `index.html` should still be the default.

File: samples/VanillaJSTestApp/index_blankPageRedirectUri.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+    }
+
+    // utils to handle standard error set that would need user interaction
+    function requiresInteraction(errorMessage) {
+        if (!errorMessage || !errorMessage.length) {
+            return false;
+        }
+
+        console.log("requiresinteraction is:" + errorMessage );
+        return errorMessage.indexOf("consent_required") !== -1 ||
```
Comment (jasonnutter) [inline samples/VanillaJSTestApp/index_blankPageRedirectUri.html]: Can we upgrade this to use the built-in static function?

File: samples/VanillaJSTestApp/server.js (hunk 3 above)
Comment (pkanher617) [reply samples/VanillaJSTestApp/server.js]: Is it possible to set up different routes? That way we don't have to keep restarting the server to test different use cases.

File: samples/VanillaJSTestApp/index_blankPageRedirectUri.html (hunk 1 above)
Comment (sameerag) [reply samples/VanillaJSTestApp/index_blankPageRedirectUri.html]: Done

File: samples/VanillaJSTestApp/index_blankPageRedirectUri.html (hunk 2 above)
Comment (sameerag) [reply samples/VanillaJSTestApp/index_blankPageRedirectUri.html]: Well, welcome message is displayed after `login` and not after `acquireTokenSilent`. I plan to change our sample structure soon; can I do it then?

File: samples/VanillaJSTestApp/server.js (hunk 3 above)
Comment (sameerag) [reply samples/VanillaJSTestApp/server.js]: Well yeah we might as well do routes soon @pkanher617, I will take it up as a separate PR, not with this one.

@jasonnutter Yes, the default should remain `index.html`, changed now.

File: samples/VanillaJSTestApp/index_blankPageRedirectUri.html (hunk 4 above)
Comment (sameerag) [reply samples/VanillaJSTestApp/index_blankPageRedirectUri.html]: yes, I will add changes with a separate PR soon. Can we delink this with the release effort?

**Pull request merged**